gruntfile.js

```diff
@@ -101,6 +101,10 @@ module.exports = function (grunt) {
         }
       }
     },
+    eslint: {
+      options: {},
```

Comment:
@lirantal we may want to add options once we finalize formatting, so I figured I would just leave it.